style.css

```diff
@@ -52,9 +52,10 @@ body {
 } */
 
 .nav {
-  display: inline-block;
-  display: flex;
+  display: flex; 
```

Comment:
Codacy found an issue: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/app.creation/pullRequest?prid=6278206)